src/App.tsx

```diff
@@ -42,10 +43,35 @@ const App: React.FC = () => {
   );
   const userId: number = parseInt(process.env.REACT_APP_USER_ID as string, 10);
   const userSignature: string = process.env.REACT_APP_USER_SIGNATURE as string;
-  const userExtend: UserSecurity = {
+  const userExtend = {
     id: userId,
     signature: userSignature,
-  };
+  } as User;
+  const UnlayerOptionsExtended = {
+    mergeTagsConfig: {
+      sort: true,
```

Comment:
What do you think about force the editor to make honor to the original order?

```suggestion
      sort: false,
```

Reply:
Corrected.

Reply:
🚀